src/client/components/AddResume/AddResume.js

```diff
@@ -24,17 +24,15 @@ const[uploadFile,setUploadaFile] =useState('');
                     <textarea type='text' value={description} onChange={(e)=> setDescription(e.target.value)}/><br/>
                 </div>
                 <div className = "upload">
-                    <label>Select A file</label><br/>
+                    <label>Select a file 
+                    <span className = "support-text"> * File supported DOC,DOCX,PDF,RTF,TXT, 5MB Max</span>
+                    </label><br/>
```

Comment:
Let's avoid the <br /> and use css instead for styling. You should be able to do a padding top on `.upload label`  i think.